caput/tests/test_mpiarray.py

```diff
@@ -20,7 +20,7 @@
 
 
 
-class TestMPIAray(unittest.TestCase):
+class TestMPIArray(unittest.TestCase):
```

Comment:
Presumably this covers redistribution? Too lazy to look.

Reply:
It does